pom.xml

```diff
@@ -225,6 +222,35 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
Add enforcer to force maven 3.6.3 or better to build.  This is adjusted from the original oss parent.